README.md

```diff
@@ -12,18 +12,59 @@ Copy `config.js.template` to `config.js`, and then fill in the variables. Finall
 
 You can enable a simple API Key verification in the `config.js` file. In this case, your requests must include the header `X-API-KEY` with the correct key.
 
-If you are using cURL to test, you can include the header using the `-H` parameter, e.g., `curl -H "X-API-Key: YourKey" http://localhost:8000/0/balance`.
+If you are using cURL to test, you can include the header using the `-H` parameter, e.g., `curl -H "X-API-Key: YourKey" http://localhost:8000/wallet/balance`.
 
 It follows the [Swagger Specification for API Keys](https://swagger.io/docs/specification/authentication/api-keys/).
 
 ## How to use?
 
 Check out the full documentation in the OpenAPI Documentation in `api-docs.js`.
 
+### Start a wallet
+
+Create and start a new wallet.
+
+Parameters:
+
+`key`: key reference of this wallet in the object of created wallets. Used to say which wallet each request is directed to.
+`passphrase`: Optional parameter to start the wallet with a passphrase. If not sent we use empty string.
+`seedKey`: Optional parameter to define which seed (from the object seeds in the config file) will be used to generate the wallet. If not sent we use 'default'.
```

Comment:
Shouldn't we allow a `seed` parameter? In this case, the API must receive either the seed or the seedKey.

Reply:
I don't know, the less we exchange seeds the better.

Reply:
I agree with @jansegre that is not safe. The advantage of allowing this is that we would be able to start new wallets dynamically, only with the seeds already added in the config file. But I guess we should discuss it more, don't know if it's a strong requirement which would make us be more flexible with security.